ckb-integration-test/src/case/rfc0031/v2019.rs

```diff
@@ -0,0 +1,57 @@
+use crate::case::rfc0031::util::test_extension_via_size;
+use crate::case::rfc0031::ERROR_UNKNOWN_FIELDS;
+use crate::case::{Case, CaseOptions};
+use crate::CKB2021;
+use ckb_testkit::Nodes;
+use ckb_testkit::{Node, NodeOptions};
+use ckb_types::core::EpochNumber;
+
+const RFC0224_EPOCH_NUMBER: EpochNumber = 3;
+
+pub struct RFC0031V2019;
+
+impl Case for RFC0031V2019 {
+    fn case_options(&self) -> CaseOptions {
+        CaseOptions {
+            make_all_nodes_connected: true,
+            make_all_nodes_synced: true,
+            make_all_nodes_connected_and_synced: true,
+            node_options: vec![NodeOptions {
+                node_name: String::from("node2021"),
+                ckb_binary: CKB2021.read().unwrap().clone(),
+                initial_database: "testdata/db/Epoch2V2TestData",
+                chain_spec: "testdata/spec/ckb2021",
+                app_config: "testdata/config/ckb2021",
+            }]
+            .into_iter()
+            .collect(),
+        }
```

Comment:
should be 2019？

Reply:
I will change the name back to `RFC0031BeforeSwitch`/`RFC0031AfterSwitch`.